packages/bot-web-ui/src/components/quick-strategy/inputs/qs-input.tsx

```diff
@@ -1,88 +1,96 @@
-                                            >
-                                                -
-                                            </button>
-                                        )
-                                    }
-                                    trailing_icon={
-                                        is_number && (
-                                            <button
-                                                data-testid='qs-input-increase'
-                                                onClick={(e: MouseEvent<HTMLButtonElement>) => {
-                                                    const value = Number(field.value) + 1;
-                                                    handleChange(e, String(value % 1 ? value.toFixed(2) : value));
-                                                }}
-                                            >
-                                                +
-                                            </button>
-                                        )
-                                    }
-                                    {...field}
-                                />
-                            </Popover>
+        const handleChange = (e: MouseEvent<HTMLButtonElement>, value: string) => {
+            e?.preventDefault();
+            setFieldTouched(name, true, true);
+            setFieldValue(name, value);
+        };
+        return (
+            <Field name={name} key={name} id={name}>
+                {({ field, meta }: FieldProps) => {
+                    const { error, touched } = meta;
+                    const has_error = error && touched;
+                    return (
+                        <div
+                            className={classNames('qs__form__field', {
+                                'full-width': fullwidth,
+                                'no-top-spacing': attached,
+                                'no-border-top': attached,
+                            })}
+                        >
+                            <div onMouseEnter={() => setFocus(true)} onMouseLeave={() => setFocus(false)}>
+                                <Popover
+                                    alignment='bottom'
+                                    message={error}
+                                    is_open={!!(error && touched && has_focus)}
+                                    zIndex='9999'
+                                    classNameBubble='qs__warning-bubble'
+                                    has_error
+                                    should_disable_pointer_events
+                                >
+                                    <Input
+                                        data_testId='qs-input'
+                                        className={classNames('qs__input', { error: has_error })}
+                                        type={type}
+                                        leading_icon={
```

Comment:
can we try to fix it for `leading_icon` and `trailing_icon` props?

This usually happens when you use conditional rendering in React and return `false` instead of component or `null`. In JSX, if you want to conditionally not render a component, the correct way to do it is to return `null` instead of `false`.

Something like this:
`is_number ? (<button ....) : null;`


<img width="982" alt="Screenshot 2023-11-02 at 11 15 28" src="https://github.com/binary-com/deriv-app/assets/103181650/7c21c625-4fa3-4946-91ab-290f334bbe85">

Reply:
I was able to fix the trailing_icon TS issue, for the leading_icon it doesn't work with null in fallback so letting it be for now, but later we will fix it or may be you can fix in your PR if possible 👍 @maryia-matskevich-deriv